reports/reporting/__init__.py

```diff
@@ -36,8 +36,7 @@ def get_report(kf_id):
 
 @reports_api.route('/<re_id>/<sd_id>', methods=['GET'])
 def get_report_per_study(re_id, sd_id):
-    endpoint_url = current_app.config['DYNAMO_ENDPOINT']
```

Comment:
I think you may need to be able to set the endpoint somehow so that you can test it using a local deployment of dynamo.